images/ec2amicreate_policy.tf

```diff
@@ -43,6 +43,18 @@ data "aws_iam_policy_document" "ec2amicreate_doc" {
 
     resources = ["*"]
   }
+
+  statement {
+    actions = [
+      "kms:Decrypt",
+      "kms:DescribeKey",
+      "kms:Encrypt",
+      "kms:GenerateDataKey*",
+      "kms:ReEncrypt*",
+    ]
+
+    resources = ["${aws_kms_key.amis.arn}"]
```

Comment:
The quotes and `${}` are unnecessary here.

Reply:
Good catch, thanks!  Fixed in https://github.com/cisagov/cool-accounts/pull/17/commits/f6f2862a843315d52de4a32f12b59ede211194a8.